docs/docs/build_and_deploy/deploy_aws.md

```
@@ -0,0 +1,91 @@
+
+### Configure Environment Variables
+
+We need to obtain the domain that the app is being deployed on: 
+- Go to the main dashboard of your app in AWS Amplify.
+- Click on `Hosting` => `Custom domains` and copy the URL to your clipboard. See the screenshot below for an example.
+
+![image](/img/custom-domains.png)
+
+Now go to `Hosting` => `Environment variables`. We need to configure two environment variables for NextAuth.js:
+- Create environment variable `NEXTAUTH_URL` and paste in the value we copied earlier from the the Custom domains screen.
+- Create environment variable `NEXTAUTH_SECRET`. The is value is used by NextAuth.js to encrypt auth cookies. See the[ NextAuth.js docs](https://next-auth.js.org/configuration/options#nextauth_secret) for creating a strong secret. You can also set this to something like "changeme" and revisit this later on, prior to launching to production.
```

Markdownlint

Spaces inside link text

(MD039, no-space-in-links)